### ISSUE TITLE
integrate with  ENHanCEd Infectious Diseases (EID2) database

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/globalbioticinteractions#339]
Re-opening of #102 as a separate issue. The #102 lead to the integration of Gemina, not ENHanCEd infectious diseases database.

**Comment by jhpoelen** [comment]
See https://eid2.liverpool.ac.uk .

**Comment by jhpoelen** [comment]
just sent email to eid2@liv.ac.uk -

----
Hi!

Some time ago (in 2015) I was made aware of the ENHanCEd Infectious Diseases (EID2) database. Some past discussions around the database in 2015 were captured in https://github.com/jhpoelen/eol-globi-data/issues/102 . In order for ENHanCEd Infectious Diseases (EID2) database to be of increased use for research, I believe that openly offering the full database as an archive is essential. If this happens, researchers no longer have to restrict their inquiries to case-by-case example, but can run analysis on the full database. Also, it would make it easier for project like Global Biotic Interactions (GloBI) to make the data more easy to find. The page at https://eid2.liverpool.ac.uk (accessed 19 Feb 2018) mentions that "Users can register free of charge here and view a limited number of pages per day." which seems to indicate that only after registration users get limited access to the database. 

I've re-opened a new discussion/issue regarding integration with your ENHanCEd Infectious Diseases (EID2) database at https://github.com/jhpoelen/eol-globi-data/issues/339 and looking forward to hearing about your plans to make your valuable database more easily/openly accessible. 

I want to stress that I acknowledge the sometimes complex political and institutional realities related to openly sharing of valuable databases. 

Curious to hear your thoughts on openly sharing your valueable ENHanCEd Infectious Diseases (EID2) database.
----

**Comment by arw36** [comment]
Just a note, as I've used this database extensively. Due to the web-scrabing method they use on NCBI (which I think is already secured into GLOBI via another pathway) and PubMed, this database can turn up many false positives (eg experimental studies, reviews, modeling papers). Not sure how that could be secured if not through manual curation.

**Comment by jhpoelen** [comment]
Hey @arw36 thanks for your comments. I'll follow your expert opinion and stop/pause pursuing the integration with ENHanCEd Infectious Diseases (EID2) by closing this issue.
